### PR TITLE
Add CONTRIBUTING.md

### LINKED ISSUE #159: Contributing.md does not exist
https://github.com/arconnectio/ArConnect/blob/development/.github/CONTRIBUTING.md is 404

### PR DESCRIPTION
**Pull request by 7i7o**
This PR updates `README.md` and adds a `CONTRIBUTING.md`

Closes #159

### CODE REVIEW

**Review by nicholaswma — APPROVED** [review]
tACK

File: .github/README.md [hunk 1]
```diff
@@ -161,14 +161,68 @@ Decrypt data with the user's wallet. This is an implementation of the [webcrypto
...
-Get the signature for a data array.
+> ⚠️ **Deprecation warning:** The `signature()` function is deprecated in ArConnect 1.0.0. Read about the alternatives below.
+
+#### Alternatives
+
+There are quite a few cases where you might need to generate a cryptographic signature for a piece of data or message so that you can verify them. The most common ones and their alternatives are the following:
+
+- Generating a signature for a transaction: [`sign()`](#signtransaction-options-promisetransaction)
+- Generating a signature for a bundle data item: [`signDataItem()`](#signdataitemdataitem) or [`dispatch()`](#dispatchtransaction-promisedispatchresult)
+- Signing a message to later validate ownership: [`signMessage()`](#signmessagedata-options-promiseuint8array) combined with [`verifyMessage()`](verify-message.md)
```
Comment (pawanpaudel93) [inline .github/README.md]: `verify-message.md` is linking to an empty md file.

Comment (7i7o) [reply .github/README.md]: Should be fixed

File: .github/README.md [hunk 2]
```diff
@@ -161,14 +161,68 @@ Decrypt data with the user's wallet. This is an implementation of the [webcrypto
...
+Requires the `SIGNATURE` [permission](#permissions).
+
+### `verifyMessage(data, signature): Promise<Boolean>`
+
+Verify validity of a cryptographic signature for a given piece of data
+
+- `data`: `ArrayBuffer` data to verify against the signature
+- `signature`: `ArrayBuffer | string` Signature to validate
+- `publicKey?`: `string` Arweave wallet `JWK.n` field, tx owner field or [public key from Arconnect](#getactivepublickey-promisestring)
+- `options`: [`SignMessageOptions`](#signmessagedata-options-promiseuint8array) Configuration for the signature
```
Comment (pawanpaudel93) [inline .github/README.md]: `SignMessageOptions` interface options seems to be missing.

Comment (7i7o) [reply .github/README.md]: Yes, It just links to SignMessage. Maybe we could link it to the docs? wdyt? I didn't want to replicate the whole docs in the readme

Comment (pawanpaudel93) [reply .github/README.md]: @7i7o Yes we could link back to docs. https://docs.arconnect.io/api/sign-message#options

Comment (7i7o) [reply .github/README.md]: Ended up adding SignMessageOptions to Readme 🤷

**Review by pawanpaudel93 — APPROVED** [review]
LGTM 🚀

**Pull request merged**